csi/shell/check_mount.sh

```diff
@@ -19,4 +21,9 @@ do
     fi
 done
 
-echo "succeed in checking mount point $ConditionPathIsMountPoint"
\ No newline at end of file
+if [ ! -e  $ConditionPathIsMountPoint/$SubPath ] ; then
+  echo "sub path [$SubPath] not exist!"
```

Comment:
I think it's better to output the full path like "`$ConditionPathIsMountPoint/$SubPath` not exist!"